sentences/en/media_player_HassSetVolume.yaml

```diff
@@ -3,6 +3,9 @@ intents:
   HassSetVolume:
     data:
       - sentences:
-          - "set <name> [volume] to <volume>"
+          - "(set|change|increase|decrease) <name> volume to <volume> [percent]"
+          - "Turn <name> volume (up|down) to <volume> [percent]"
+          - "((set|change|increase|decrease) the volume to <volume> [percent];[on ]<name>)"
+          - "(Turn the volume (up|down) to <volume> [percent];[on ]<name>)"
```

Comment:
```suggestion
          - "(turn the volume (up|down) to <volume>;[on ]<name>)"
```

Reply:
Consider using `(the volume;(up|down))`, as both
`turn up the volume to 12 on X`
`turn the volume up to 12 on X`
could work